### PR TITLE
Translate missing pages for application faq section

### PR DESCRIPTION
**Pull request by Arthur-GYT**
All is in the title ^^.

### CODE REVIEW

File: content/faq/app/how-to-use-android-auto/index.fr.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Comment utiliser Android Auto ?
+description: Configuration minimale requise pour utiliser Organic Maps avec Android Auto
+slug: comment utiliser-android-auto
```
Comment (biodranik) [inline content/faq/app/how-to-use-android-auto/index.fr.md:4]:
```suggestion
slug: comment-utiliser-android-auto
```

**Review by biodranik — APPROVED** [review]
Merci !

File: content/faq/app/how-to-use-android-auto/index.fr.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+slug: comment-utiliser-android-auto
+
+taxonomies:
+  faq: ["Application"]
+
+extra:
+  order: 20
+---
+
+Pour utiliser Organic Maps avec Android Auto, vous avez besoin au minimum d'une version Android 8.0 (Oreo) ou ultérieure.
```
Comment (Jean-BaptisteC) [inline content/faq/app/how-to-use-android-auto/index.fr.md:13]:
```suggestion
Pour utiliser Organic Maps sur Android Auto, vous avez besoin d'un appareil avec au minimum Android 8.0 (Oreo) ou ultérieure.
```

File: content/faq/app/which-ios-and-android-versions-are-supported/index.fr.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+slug: quelles-versions-ios-et-android-sont-supportées
+
+taxonomies:
+  faq: ["Application"]
+
+extra:
+  order: 10
+---
+
+Vous devez avoir au moins iOS 12 ou Android 5 pour utiliser l'application. Les appareils dotés de versions ultérieures peuvent utiliser Organic Maps.
```
Comment (Jean-BaptisteC) [inline content/faq/app/which-ios-and-android-versions-are-supported/index.fr.md:13]:
```suggestion
Vous devez avoir au mininum iOS 12 ou Android 5 sur votre appareil pour utiliser l'application. Les appareils dotés de versions ultérieures peuvent utiliser Organic Maps.
```

File: content/faq/app/which-ios-and-android-versions-are-supported/index.fr.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+In Android, Organic Maps can work on devices with Google Services installed and without Google support.
```
Comment (Jean-BaptisteC) [inline content/faq/app/which-ios-and-android-versions-are-supported/index.fr.md:15]:
```suggestion
Sur Android, Organic Maps peut fonctionner sur des appareils avec ou sans les Services Google Play d'installées.
```

File: content/faq/app/which-ios-and-android-versions-are-supported/index.fr.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+Pour utiliser Organic Maps avec Android Auto, veuillez vérifier les conditions requises [ici](../how-to-use-android-auto/).
```
Comment (Jean-BaptisteC) [inline content/faq/app/which-ios-and-android-versions-are-supported/index.fr.md:17]:
```suggestion
Pour utiliser Organic Maps sur Android Auto, veuillez vérifier les prérequis [ici](../how-to-use-android-auto/).
```

**Review by Jean-BaptisteC — COMMENTED** [review]
Somes changes is required

**Pull request merged**